:hugo-runtime:install is failing

I made a clone of the project and then I built it. When I try to run the install task this occurs. maven is installed and working so I am not sure what's causing it.
:hugo-runtime:install FAILED          
               
FAILURE: Build failed with an exception.
               
* What went wrong:
Execution failed for task ':hugo-runtime:install'.
> A problem occurred starting process 'command 'mvn''


This is the stacktrace

```
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':hugo-runtime
:install'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute
Actions(ExecuteActionsTaskExecuter.java:69)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute
(ExecuteActionsTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExecuter.
execute(PostExecutionAnalysisTaskExecuter.java:35)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(S
kipUpToDateTaskExecuter.java:64)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(Val
idatingTaskExecuter.java:58)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.e
xecute(SkipEmptySourceFilesTaskExecuter.java:42)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.exec
ute(SkipTaskWithNoActionsExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(Ski
pOnlyIfTaskExecuter.java:53)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.exec
ute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.AbstractTask.executeWithoutThrowingTaskFailure(Abs
tractTask.java:305)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker
.executeTask(AbstractTaskPlanExecutor.java:79)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker
.processTask(AbstractTaskPlanExecutor.java:63)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker
.run(AbstractTaskPlanExecutor.java:51)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor.process(DefaultTask
PlanExecutor.java:23)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter.execute(DefaultTas
kGraphExecuter.java:88)
        at org.gradle.execution.SelectedTaskExecutionAction.execute(SelectedTaskExecu
tionAction.java:29)
        at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.jav
a:62)
        at org.gradle.execution.DefaultBuildExecuter.access$200(DefaultBuildExecuter.
java:23)
        at org.gradle.execution.DefaultBuildExecuter$2.proceed(DefaultBuildExecuter.j
ava:68)
        at org.gradle.execution.DryRunBuildExecutionAction.execute(DryRunBuildExecuti
onAction.java:32)
        at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.jav
a:62)
        at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.jav
a:55)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradl
eLauncher.java:149)
        at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLaunc
her.java:106)
        at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.
java:86)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildControll
er.run(InProcessBuildActionExecuter.java:80)
        at org.gradle.launcher.cli.ExecuteBuildAction.run(ExecuteBuildAction.java:33)

        at org.gradle.launcher.cli.ExecuteBuildAction.run(ExecuteBuildAction.java:24)

        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBui
ldActionExecuter.java:36)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBui
ldActionExecuter.java:26)
        at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:51)
        at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:171
)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execu
te(CommandLineActionFactory.java:237)
        at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execu
te(CommandLineActionFactory.java:210)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeVal
idationAction.java:35)
        at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeVal
idationAction.java:24)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(Comma
ndLineActionFactory.java:206)
        at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(Comma
ndLineActionFactory.java:169)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportin
gAction.java:33)
        at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportin
gAction.java:22)
        at org.gradle.launcher.Main.doAction(Main.java:33)
        at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.
java:54)
        at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:3
5)
        at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
Caused by: org.gradle.process.internal.ExecException: A problem occurred starting pro
cess 'command 'mvn''
        at org.gradle.process.internal.DefaultExecHandle.setEndStateInfo(DefaultExecH
andle.java:196)
        at org.gradle.process.internal.DefaultExecHandle.failed(DefaultExecHandle.jav
a:325)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:83)

        at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImp
l$1.run(DefaultExecutorFactory.java:64)
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'mvn'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultP
rocessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WindowsProcessLauncher.start(WindowsP
rocessLauncher.java:22)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperP
rocessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:65)

        ... 1 more
Caused by: java.io.IOException: Cannot run program "mvn" (in directory "C:\Users\Joel
\Documents\Android Projects\hugo\hugo-runtime"): CreateProcess error=2, The system ca
nnot find the file specified
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultP
rocessLauncher.java:25)
        ... 4 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the fil
e specified
        ... 5 more
```

--- a/build.gradle
+++ b/build.gradle
@@ -8,6 +8,7 @@ buildscript {
     classpath 'org.gradle.api.plugins:gradle-nexus-plugin:0.7'
     classpath 'com.android.tools.build:gradle:1.1.0-rc3'
     classpath 'org.aspectj:aspectjtools:1.8.5'
+    classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
   }
 }
 

--- a/build.gradle
+++ b/build.gradle
@@ -6,8 +6,8 @@ buildscript {
 
   dependencies {
     classpath 'org.gradle.api.plugins:gradle-nexus-plugin:0.7'
-    classpath 'com.android.tools.build:gradle:1.1.0-rc3'
-    classpath 'org.aspectj:aspectjtools:1.8.5'
+    classpath 'com.android.tools.build:gradle:1.3.0'
+    classpath 'org.aspectj:aspectjtools:1.8.6'
     classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
   }
 }

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.5-all.zip

--- a/hugo-example/build.gradle
+++ b/hugo-example/build.gradle
@@ -7,7 +7,7 @@ buildscript {
       mavenLocal()
     }
 
-    classpath 'com.android.tools.build:gradle:1.1.0-rc3'
+    classpath 'com.android.tools.build:gradle:1.3.0'
     classpath 'com.jakewharton.hugo:hugo-plugin:1.2.2-SNAPSHOT'
   }
 }

--- a/hugo-plugin/build.gradle
+++ b/hugo-plugin/build.gradle
@@ -8,9 +8,9 @@ sourceCompatibility = JavaVersion.VERSION_1_7
 dependencies {
   compile gradleApi()
   compile localGroovy()
-  compile 'com.android.tools.build:gradle:1.1.0-rc3'
-  compile 'org.aspectj:aspectjtools:1.8.5'
-  compile 'org.aspectj:aspectjrt:1.8.5'
+  compile 'com.android.tools.build:gradle:1.3.0'
+  compile 'org.aspectj:aspectjtools:1.8.6'
+  compile 'org.aspectj:aspectjrt:1.8.6'
 }
 
 modifyPom {

--- a/hugo-plugin/src/main/groovy/hugo/weaving/plugin/HugoPlugin.groovy
+++ b/hugo-plugin/src/main/groovy/hugo/weaving/plugin/HugoPlugin.groovy
@@ -28,7 +28,7 @@ class HugoPlugin implements Plugin<Project> {
     project.dependencies {
       debugCompile 'com.jakewharton.hugo:hugo-runtime:1.2.2-SNAPSHOT'
       // TODO this should come transitively
-      debugCompile 'org.aspectj:aspectjrt:1.8.5'
+      debugCompile 'org.aspectj:aspectjrt:1.8.6'
       compile 'com.jakewharton.hugo:hugo-annotations:1.2.2-SNAPSHOT'
     }
 

--- a/hugo-runtime/build.gradle
+++ b/hugo-runtime/build.gradle
@@ -1,10 +1,6 @@
-import com.android.build.gradle.LibraryPlugin
 import org.aspectj.bridge.IMessage
 import org.aspectj.bridge.MessageHandler
 import org.aspectj.tools.ajc.Main
-
-import static org.apache.tools.ant.taskdefs.condition.Os.FAMILY_WINDOWS
-import static org.apache.tools.ant.taskdefs.condition.Os.isFamily
 
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
@@ -27,7 +23,6 @@ android {
 }
 
 android.libraryVariants.all { variant ->
-  LibraryPlugin plugin = project.plugins.getPlugin(LibraryPlugin)
   JavaCompile javaCompile = variant.javaCompile
   javaCompile.doLast {
     String[] args = [

--- a/hugo-runtime/build.gradle
+++ b/hugo-runtime/build.gradle
@@ -7,6 +7,7 @@ import static org.apache.tools.ant.taskdefs.condition.Os.FAMILY_WINDOWS
 import static org.apache.tools.ant.taskdefs.condition.Os.isFamily
 
 apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
 
 dependencies {
   compile 'org.aspectj:aspectjrt:1.8.5'
@@ -61,23 +62,3 @@ android.libraryVariants.all { variant ->
     }
   }
 }
-
-// Use Maven to install the aar into our local maven repo. Quick and easy (and dirty).
-task install(type: Exec, dependsOn: assemble) {
-  // TODO get aar path dynamically when i'm not being lazy
-  if (isFamily(FAMILY_WINDOWS)) {
-    executable = 'mvn.bat'
-  } else{
-    executable = 'mvn'
-  }
-  args = [
-      "install:install-file",
-      "-DgroupId=com.jakewharton.hugo",
-      "-DartifactId=hugo-runtime",
-      "-Dversion=${version}",
-      "-Dpackaging=aar",
-      "-Dfile=build/outputs/aar/hugo-runtime-release.aar"
-  ]
-}
-
-apply from: 'https://raw.github.com/chrisbanes/gradle-mvn-push/master/gradle-mvn-push.gradle'

--- a/hugo-runtime/build.gradle
+++ b/hugo-runtime/build.gradle
@@ -10,7 +10,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
 
 dependencies {
-  compile 'org.aspectj:aspectjrt:1.8.5'
+  compile 'org.aspectj:aspectjrt:1.8.6'
   compile project(':hugo-annotations')
 
   testCompile 'junit:junit:4.12'

